### DISCUSSION TITLE
Why did Lossless Cut shift from a single .exe to a .7z for Windows?

### DISCUSSION THREAD

**Discussion by Welding-Torch** [mifi/lossless-cut#1970] [Q&A] [answered]
Hi there 👋

So I have an older version of Loseless Cut (v42) and I have it downloaded as a single, portable executable file. This is very nice and convenient. However, when I downloaded the latest version of Lossless Cut today, I noticed that it's only offered as a .7z. This is slightly less convenient as it involves unzipping. 

May I know why this change happened? 

Thanks.

**Comment by mifi** [comment, marked as answer]
Hi. please see faq and help page: https://github.com/mifi/lossless-cut/blob/3f4c2142879cb28670a131aea49e80e500d55d64/issues.md?plain=1#L104

**Reply by Welding-Torch** [reply]
Thanks, this cleared it up 👍
